components/checkbox/stories/template.js

```diff
@@ -90,3 +90,68 @@ export const Template = ({
 		</label>
 	`;
 };
+
+export const CheckboxGroup = (args, context) => html`
+	<div style="display: flex; flex-direction: column; padding: 16px">
+		${Template({
+			...args,
+			context,
+			iconName: undefined,
+		})}
+		${Template({
+			...args,
+			context,
+			isChecked: true,
+		})}
+		${Template({
+			...args,
+			context,
+			isIndeterminate: true,
+		})}
+		${Template({
+			...args,
+			context,
+			isDisabled: true,
+		})}
+		${Template({
```

Comment:
Should we test truncation separately from other content variations? They seem kind of distinct and they can sometimes bloat a grouping. Do we miss out on anything by testing it separately?

Reply:
I pulled this out and [put it in the testing grid](https://github.com/adobe/spectrum-css/pull/2793/commits/a3305ea61c0391796d3c836af336abc705979d37) for checkbox instead. Does that work?
<img width="893" alt="Screenshot 2024-08-12 at 3 49 53 PM" src="https://github.com/user-attachments/assets/05e624be-3295-47eb-86d7-afcebc71da54">

I reverted the example in `CheckboxGroup` to something more akin to how it was before, with a much shorter, wrapping label. I think it's good to show the wrapping behavior on the docs page, especially since I didn't create a specific "wrapping" story to mimic the static docs site (which has a totally separate variant for wrapping behavior).
<img width="872" alt="Screenshot 2024-08-12 at 3 50 46 PM" src="https://github.com/user-attachments/assets/660c3fbf-3d6e-4881-8176-8c59140a7fbd">